docs/NativeCodeGeneration.md

```diff
@@ -109,12 +113,12 @@ typedef struct _program_entry
 } program_entry_t;
```

Comment:
This is the updated `program_entry_t`

`typedef struct _program_entry
    {
        uint64_t (*function)(void*);
        const char* section_name;
        const char* program_name;
        uint16_t* referenced_map_indices;
        uint16_t referenced_map_count;
        helper_function_entry_t* helpers;
        uint16_t helper_count;
        size_t bpf_instruction_count;
        ebpf_program_type_t* program_type;
        ebpf_attach_type_t* expected_attach_type;
    } program_entry_t;`

Reply:
Thanks, updated.